### ISSUE TITLE
Define GitHub action to create auto commits

### ISSUE THREAD

**Issue by arsulegai** [hyperledger-tooling/github-updates#3]
GitHub action will run the tool at regular interval, create the commits and update to the repository.
1. GitHub action shall auto identify the diff from previous run.
2. Create a commit and push to the repository.

**Comment by arsulegai** [comment]
Defined the GitHub action that will `autocommit` the PRs labelled as `automerge`. More info: https://github.com/arsulegai/start-here-hyperledger/commit/3d5cc6ab83610c7221ae32e24fe77a82d6fd5d82

**Comment by ryjones** [comment]
@arsulegai this might be of interest: https://github.com/hyperledger-landscape/hl-landscape/tree/master/.github/workflows

**Comment by arsulegai** [comment]
Thanks @ryjones. Any suggestions on which theme to use? Waiting on the theme to finalize the output format and add this plugin.

**Comment by ryjones** [comment]
I use the minimal theme. Look at the source for the TSC repo: https://github.com/hyperledger/tsc/blob/d4c96e355d6c9958a8dd6d87259586ee1f4a0514/_config.yml#L2

**Comment by arnabkaycee** [comment]
Since the Jekyll page is set up, I can help out with generating the MD files for releases, issues and PRs. Can we create a new issue to define the structure of the template of the MD that gets auto committed?

**Comment by arsulegai** [comment]
@arnabkaycee @ryjones here is the site up and running https://arsulegai.github.io/start-here-hyperledger/ .
@arnabkaycee feel free to add enhancements.
